src/trackinfoobject.h

```diff
@@ -32,10 +32,27 @@ class TrackInfoObject : public QObject {
     Q_OBJECT
 
   public:
-    // Initialize track with a QFileInfo class
-    explicit TrackInfoObject(
+    TrackInfoObject(const TrackInfoObject&) = delete;
+    TrackInfoObject(TrackInfoObject&&) = delete;
+
+    // Creates a new empty temporary instance for fake tracks or for
+    // testing purposes. The resulting track will neither be stored
+    // in the database nor will the metadata of the corresponding file
+    // be updated.
+    static TrackPointer newTemporary(
             const QFileInfo& fileInfo = QFileInfo(),
-            SecurityTokenPointer pToken = SecurityTokenPointer());
+            const SecurityTokenPointer& pSecurityToken = SecurityTokenPointer());
+    // Creates a new temporary instance from another track that references
+    // the same file. Please remember that the corresponding file might
+    // be modified after all references to the original track have been
+    // dropped. To be safe just keep the pointer to the original track
```

Comment:
I do not understand why this is a problem. If the original track looses all references it is no needed any more, so what bad could happen than?